libs/sdk-ui-pivot/src/impl/AggregationsMenu.tsx

```diff
@@ -133,12 +134,13 @@ export default class AggregationsMenu extends React.Component<IAggregationsMenuP
         onClick: () => void,
         isSelected: boolean,
         hasSubMenu = false,
-        isEnabled: boolean,
+        isDisabled: boolean,
```

Comment:
small stuff: consider renaming to `disabled` - better match with the `Item` prop